infoFileComparison/src/main/scala/za/co/absa/hermes/infoFileComparison/InfoFileComparisonJob.scala

```diff
@@ -0,0 +1,102 @@
+package za.co.absa.hermes.infoFileComparison
+
+import java.io.FileInputStream
+
+import better.files.File
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.commons.io.IOUtils
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.permission.FsPermission
+import org.apache.hadoop.fs.{FileSystem, Path}
+import za.co.absa.atum.model.ControlMeasure
+import za.co.absa.atum.persistence.ControlMeasuresParser
+import za.co.absa.atum.utils.ARMImplicits
+import za.co.absa.hermes.infoFileComparison.AtumModelUtils._
+
+import scala.collection.JavaConverters._
+
+object InfoFileComparisonJob {
+  private val conf: Config = ConfigFactory.load
+  private val bufferSizeDefaultValue = conf.getInt("info-file-comparison.comparison-job.bufferSizeDefaultValue")
+
+  private lazy val hadoopConfiguration = getHadoopConfiguration
+
+  def main(args: Array[String]): Unit = {
+    val cmd = InfoComparisonConfig.getCmdLineArguments(args)
+
+    execute(cmd)
+  }
+
+  /**
+    * Execute the comparison
+    *
+    * @param cmd Provided configuration for the comparison
+    */
+  def execute(cmd: InfoComparisonConfig): Unit = {
+    val newControlMeasure = loadControlMeasures(cmd.newPath)
+    val refControlMeasure = loadControlMeasures(cmd.refPath)
+
+    val diff: List[ModelDifference[_]] = refControlMeasure.compareWith(newControlMeasure)
+
+    if (diff.nonEmpty) {
+      val serializedData: String = ModelDifferenceParser.asJson(diff)
```

Comment:
```suggestion
      val serializedData = ModelDifferenceParser.asJson(diff)
```
Matter of preference, up to you